.github/workflows/push.yaml

```diff
@@ -53,3 +58,32 @@ jobs:
         with:
           name: redskyctl_linux_amd64
           path: dist/redskyctl-linux-amd64.tar.gz
+      - name: redsky quickstart
+        run: |
+          echo "Init redskyops"
+          dist/redskyctl_linux_amd64/redskyctl init
+          echo "Install kustomize"
+          curl -s "https://raw.githubusercontent.com/\
+          kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | \
+            bash
+          echo "Patch postgres deployment"
+          ./kustomize build github.com/redskyops/redskyops-recipes/postgres > postgres.yaml
+          # Need to change strategy here since we're limited to 2cpu total
+          sed -i -e '37i\  strategy:\n    type: Recreate' postgres.yaml
```

Comment:
My vote here would be to create `/test/quickstart/kustomization.yaml` file:

```yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

resources:
- github.com/redskyops/redskyops-recipes/postgres

patchesStrategicMerge:
- |-
  apiVersion: apps/v1
  kind: Deployment
  metadata:
    name: postgres
  spec:
    strategy:
      type: Recreate
```
That way you can just `kustomize build test/quickstart  | kubectl apply -f -`.

I'm on the fence about adding a `/test/quickstart/run.sh` to do everything from here down.